vision/transforms/permute.go

```diff
@@ -0,0 +1,22 @@
+package transforms
+
+import torch "github.com/wangkuiyi/gotorch"
+
+// PermuteTransformer permute the input tensor to desired
+type PermuteTransformer struct {
```

Comment:
Maybe we don't need this Transformer, in PyTorch the image would be transposed to `CHW` format by default.
c.f. https://github.com/pytorch/vision/blob/fc69c22576cbccb59c581ddbaca4dedbdb279688/torchvision/transforms/functional.py#L123